go-controller/pkg/ovn/ovn.go

```diff
@@ -478,15 +482,39 @@ func (oc *Controller) recordPodEvent(addErr error, pod *kapi.Pod) {
 	}
 }
 
+const retryInitialDelay time.Duration = 2 * time.Second
+
+func newRetryEntry(pod *kapi.Pod, ignore bool) *retryEntry {
+	return &retryEntry{
+		pod:        pod,
+		timeStamp:  time.Now().Add(retryInitialDelay),
```

Comment:
it looks like time stamp was originally the time when the pod was added to the retry cache, but now it is the time when we are going to retry this pod. would be good to document this in the struct.

The retryPods map also has a comment that says: "// Map of pods that need to be retried, and the timestamp of when they last failed"

^this is no longer accurate

Reply:
done